### PR TITLE
Update the expand hub template access design

### PR DESCRIPTION
**Pull request by mprahl**
Relates:
https://issues.redhat.com/browse/ACM-12129

### CODE REVIEW

File: enhancements/sig-policy/122-expanded-hub-templates-access/README.md [hunk 1]
```diff
@@ -271,8 +289,25 @@ func (r *ReplicatedPolicyReconciler) getToken(ctx context.Context, namespace str
 }
 ```
 
+#### Permission Changes to the Service Account
+
+When permissions change after a watch has started, authorization is not reassessed until there is a new request, even
+when the token expires. The [kubernetes-dependency-watches](https://github.com/stolostron/kubernetes-dependency-watches)
+library leverages the client-go [RetryWatcher](https://pkg.go.dev/k8s.io/client-go/tools/watch#RetryWatcher) and it does
+not currently error if the user does not have `watch` permissions. For when the watch is first started,
```
Comment (yiraeChristineKim) [inline enhancements/sig-policy/122-expanded-hub-templates-access/README.md:297]:
```suggestion
not currently have an error  if the user does not have `watch` permissions. For when the watch is first started,
```

**Review by yiraeChristineKim — APPROVED** [review]

**Review by qiujian16 — COMMENTED** [review]
/lgtm

**Comment by mprahl** [comment]
@yiraeChristineKim I fixed a typo. Could you please take another look?

**Review by yiraeChristineKim — APPROVED** [review]

**Pull request merged**